_includes/header.html

```diff
@@ -16,10 +16,10 @@
     </li>
     <li class="col-xs-12 col-md-10 menu">
       <nav aria-label="Banner links">
-        <h2><a href="/GettingStarted">Getting Started</a></h2>
+        <h2><a href="/Resources">Resources</a></h2>
         <h2><a href="/processes">Contributing</a></h2>
         <h2><a href="/projects">Projects</a></h2>
-        <h2><a href="/Tutorials">Tutorials</a></h2>
+        <h2><a href="/tutorials/">Tutorials</a></h2>
```

Comment:
So this change doesn't appear for me on the file system -- the directory there is called `Tutorials` with capital `T` for me, and the links to lower-case `tutorials` are not working because of that.

Which one is the intended one? (The Mac filesystem is case insensitive, so my repo checkout would not have seen a rename from the `Tutorials` directory to `tutorials`, it'd have stayed the same).

Reply:
Looking into it further, it gets introduced in [this commit](https://github.com/seL4/docs/commit/68c00ea5893404402807c44fe599f4e2d5334aae) which might mean it was an attempt to get the link rendering working correctly. The rename `Tutorials` -> `tutorials` should be reverted, though.